### PR TITLE
Foundations - Javascript: Add note about package warnings

### PR DESCRIPTION
**Pull request by fabulousgk**
## Because

The packages for the exercises get out of date and produce warnings when running `npm install`. This confuses new learners.

## This PR

- Add a note that the warnings can be safely ignored

## Issue

Related to: https://github.com/TheOdinProject/javascript-exercises/issues/250

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/javascript_basics/fundamentals-4.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ Follow the steps below to get started. Once you complete Step 1, be sure to use
     * tempConversion
 3. Once you have a working solution, see how it compares to the exercise's given solution. Solutions for the exercises can be found in the 'solution' folder of each exercise.
 
+(Note: if you get warnings that packages are out of date or contain vulnerabilities when running `npm install`, you can safely ignore them for these exercises.)
```
Comment (KevinMulhern) [inline foundations/javascript_basics/fundamentals-4.md:55]: Thanks for doing this, Can we use [one of our note boxes](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) please?

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: foundations/javascript_basics/fundamentals-4.md (hunk 1 above)
Comment (fabulousgk) [reply foundations/javascript_basics/fundamentals-4.md:55]: Sure thing, I was not sure exactly which one so I went with the standard.

**Comment by ManonLef** [comment]
I'll let @KevinMulhern review this since he requested the changes and didn't address the fact It's in the lesson. It makes me consider if there was some possible discussion preceding this decision that I'm unaware of. 

My thoughts: since the installation/setup instructions are in the readme of the exercise repo, why wouldn't we put this note in there?

If there was a decision made to put it in the lesson instead, I believe the note should be added under point 2 or right before it. Note 2 is also worded a little strange `Now that you have cloned the repository and run npm install to install Jest` should probably be `ran`. 

sidenote: The backticks inside the note don't work. You'd probably have to wrap it inside a `<code>` element.

**Comment by fabulousgk** [comment]
There is a separate PR for the readme.  I added the note here because of the mention of npm install, in case someone did not actually read the README.  Based on these comments though, perhaps the better solution here is just to update the text of bullet point #2. Perhaps something like:

Complete these exercises in the following order (the remaining exercises will be completed later.) Make sure to review **each exercise’s README** for instructions:

**Comment by KevinMulhern** [comment]
Thanks @fabulousgk, having this on the README should hopefully be enough for now. We can reassess and add this in later if learners miss it.

**Comment by ManonLef** [comment]
Sorry to comment here again @KevinMulhern but I believe you may have thought @fabulousgk was proposing to add the npm remark in the lesson AND the exercise readme:

I believe what what they are proposing instead is to *also* rewrite point 2 in the lesson which I think is a great idea due to the concerns I raised about it here: https://github.com/TheOdinProject/curriculum/pull/26213#issuecomment-1693434556

> Now that you have cloned the repository and run npm install to install Jest

☝️ that is included in point 2 of the exercise is now. Which is a bit silly since all instructions have already been followed in point 1.

**Comment by fabulousgk** [comment]
I deleted the branch but can easily do a new PR if everyone is in agreement.

**Pull request closed without merging**